Client/Frontend/Widgets/PhotonActionSheet.swift

```diff
@@ -39,18 +39,20 @@ public struct PhotonActionSheetItem {
     public fileprivate(set) var text: String?
     public fileprivate(set) var iconString: String?
     public fileprivate(set) var iconURL: URL?
+    public fileprivate(set) var iconType: PhotonActionSheetIconType
     public fileprivate(set) var iconAlignment: IconAlignment
 
     public var isEnabled: Bool // Used by toggles like nightmode to switch tint color
     public fileprivate(set) var accessory: PhotonActionSheetCellAccessoryType
     public fileprivate(set) var accessoryText: String?
     public fileprivate(set) var bold: Bool = false
     public fileprivate(set) var handler: ((PhotonActionSheetItem) -> Void)?
-
-    init(title: String, text: String? = nil, iconString: String? = nil, iconURL: URL? = nil, iconAlignment: IconAlignment = .left, isEnabled: Bool = false, accessory: PhotonActionSheetCellAccessoryType = .None, accessoryText: String? = nil, bold: Bool? = false, handler: ((PhotonActionSheetItem) -> Void)? = nil) {
+    
+    init(title: String, text: String? = nil, iconString: String? = nil, iconURL: URL? = nil, iconType: PhotonActionSheetIconType = .URL, iconAlignment: IconAlignment = .left, isEnabled: Bool = false, accessory: PhotonActionSheetCellAccessoryType = .None, accessoryText: String? = nil, bold: Bool? = false, handler: ((PhotonActionSheetItem) -> Void)? = nil) {
```

Comment:
The default type for `iconType` should be image as I think we use that the most.

Reply:
fixed